tests/behat/fixtures/merlin_ui_extracted_content.json

```diff
@@ -0,0 +1,22 @@
+[
```

Comment:
please remove `merlin ui` from the title

Please name this `civictheme_migrate.page_content_1.json`